python/lsst/sconsUtils/installation.py

```diff
@@ -95,6 +95,13 @@ def setPrefix(env, versionString, eupsProductPath=None):
                 "%s\nFound problem with version number; update or specify force=True to proceed"
                 % err
             )
+
+    if state.env['no_eups']:
+        if 'prefix' in env and env['prefix']:
+            return env['prefix']
+        else:
+            return "/usr/local"
+
```

Comment:
Its also the current default behavior, the branch I inserted just bails out of the method before reaching it.  https://github.com/lsst/sconsUtils/blob/tickets/DM-2769/python/lsst/sconsUtils/installation.py#L127  This isn't very DRY but I was trying not to rework the existing logic.